app/models/comment.php

```diff
@@ -75,4 +144,165 @@ public function edit($comment_id)
             $db->rollback();
         }
     }
+
+    //delete all comments that has the given thread_id
+    public function deleteComments($thread_id)
+    {
+        try {
+            $db = DB::conn();
+            $db->begin();
+
+            $delete = $db->query('DELETE FROM comment WHERE thread_id = ?', array($thread_id));
+
+            $db->commit();
+        } catch (Exception $e) {
+            $db->rollback();
+        }
+    }
+
+    //delete one comment
+    public function deleteComment($comment_id)
```

Comment:
Since this class is class is already named "Comment", you can remove redundancy by renaming this function to "delete", and renaming the deleteComments to "deleteAll" :bow: